shpc/main/container/config.py

```diff
@@ -259,26 +304,29 @@ def get_envars(self):
         """
         return dict(self.env) if self.env else {}
 
-    def get_aliases(self):
+    def get_aliases(self, tag=None):
```

Comment:
This might be a very dumb question: why is this function changed ? I can't see the `tag` parameter being used, and I don't understand the renames `aliases` → `loaded` and `self.aliases` → `aliases`.
Is it a leftover from the previous version where only aliases could be overridden ?

Reply:
This tag=None should be removed - that was an oversight! When the changes were scoped to aliases, we had lines here that would essentially load the file (and aliases) on demand. Now as overrids, that happens once at the beginning of the install (so we don't need the tag here). It was my oversight to not remove that (now unneeded) tag parameter.